File: core/src/main/java/me/devsnox/planetsystem/core/database/PlayerDAO.java
package me.devsnox.planetsystem.core.database;

import org.mongodb.morphia.Datastore;
import org.mongodb.morphia.dao.BasicDAO;

public class PlayerDAO extends BasicDAO<DatabasePlayer, String> {

    public PlayerDAO(Class<DatabasePlayer> entityClass, Datastore ds) {
        super(entityClass, ds);
    }
}

File: core/src/main/kotlin/de/astride/planetsystem/core/database/PlayerDAO.kt
/*
 * © Copyright - Astride UG (haftungsbeschränkt) 2018 - 2019.
 */

package de.astride.planetsystem.core.database

import de.astride.planetsystem.core.database.entities.BasicOfflinePlayer
import xyz.morphia.Datastore
import xyz.morphia.dao.BasicDAO

class PlayerDAO(entityClass: Class<BasicOfflinePlayer>, ds: Datastore) :
    BasicDAO<BasicOfflinePlayer, String>(entityClass, ds)
